deploy-manage/cloud-connect.md

```diff
@@ -0,0 +1,58 @@
+---
+applies_to:
+  deployment:
+    self:
+navigation_title: Cloud Connect
+---
+
+# Cloud Connect
+
+Cloud Connect enables you to use {{ecloud}} services in your self-managed cluster without having to install and maintain their infrastructure yourself. In this way, you can get faster access to new features while significantly reducing your operational overhead.
+
+Cloud Connect is included with your [self-managed Enterprise license](https://www.elastic.co/subscriptions) and works with self-managed free trials for the duration of the trial period.
+
+## Available cloud connected services 
+
+The following service is currently available for use with Cloud Connect. More services are coming soon.
+
+
+### AutoOps
```

Comment:
I know this structure will have to change when EIS is available, but we shouldn't have a list of one. we should just say in the sentence above that autoops is the only available service and skip the heading.